### DISCUSSION TITLE
Checkout: Free shipping label

### DISCUSSION THREAD

**Discussion by josias-hosang** [swellstores/community#441] [Ideas]
The label in the checkout overview where to select the shipping should be a translatable field. If there are no shipping costs per configuration, it's written "FREE" which is not a localized string. Field should be added in Storefronts > Checkout > Customize checkout > Language
<img width="754" alt="Screenshot 2023-11-14 at 17-38-26 Checkout" src="https://github.com/swellstores/community/assets/48205514/fbb29c26-d030-404d-90b0-8f83e79e2336">

**Comment by ericingram** [comment]
Good point!
